Hippo.Core/Services/NotificationService.cs

```diff
@@ -14,13 +14,16 @@
 using Razor.Templating.Core;
 using Mjml.Net;
 using Hippo.Core.Models;
+using StrawberryShake.Json;
```

Comment:
What's this for? Isn't it a graphQL thing?

Reply:
Not sure how that got in there. Cleaning it up...

Reply:
We do ChartString validation, but I don't think that is needed there.